src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -137,6 +137,16 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 	}
 
 	ngOnInit() {
+		/// For focusing the correct cell on tabbing, we hook into the focus_in and check if the active cell
+		/// is parent of the tabbed into element.
+		this._register(DOM.addDisposableListener(window, DOM.EventType.FOCUS_IN, () => {
```

Comment:
We shouldn't need to set this on the entire window - just doing the container element here should be enough